geist-mono/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "geist-mono",
+  "family": "Geist Mono",
+  "subsets": ["latin"],
+  "weights": [100, 200, 300, 400, 500, 600, 700, 800, 900],
+  "styles": ["normal"],
+  "defSubset": "latin",
+  "variable": false,
+  "lastModified": "2023-12-19",
+  "version": "v1.1.0",
+  "category": "monospace",
+  "license": {
+    "type": "OFL-1.1",
+    "url": "https://openfontlicense.org/",
```

Comment:
Should update this to [Github link](https://github.com/vercel/geist-font/blob/main/LICENSE.TXT).